services/offline/src/lib/__tests__/offline-provider.test.tsx

```diff
@@ -71,7 +72,9 @@ describe('Testing offline provider', () => {
         const { getByTestId } = screen
         expect(testOfflineInterface.getCachedSections).toHaveBeenCalled()
         await waitFor(() => getByTestId('sections').textContent !== '{}')
-        const textContent = JSON.parse(getByTestId('sections').textContent)
+        const textContent = JSON.parse(
+            getByTestId('sections').textContent || ''
```

Comment:
It's weird that this is needed since `getByTestId` either [returns a html element](https://github.com/testing-library/dom-testing-library/blob/d8a8d581b03fafb3dbee060ef7d5cbfe991aaf08/types/queries.d.ts#L24-L28) or throws an error, and every html element should have a `textContent` attribute :thinking:

Reply:
I think it's because `Node.textContent` [can be either a string or `null`](https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#value)